### DISCUSSION TITLE
Allow adding multiple modules at once

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [nuxt/cli#246] [Ideas]
Today I made a new project, a full stack Nuxt chat, add tried to install multiple modules with `pnpm dlx nuxi@latest module add nuxt-quasar-ui nuxt/i18n @pinia/nuxt nuxt-lodash nuxt-mongoose` and only the first one was installed and added to the modules array in nuxt.config.ts.

I'm so used to being able to install multiple Node packages with `pnpm add package1 package 2 package3` that 
i did it automatically.

It would be nice to able to do this. I've been playing with nuxi add so I can contribute extending it, but I ended creating my own scaffolding tool using inquirer for the console UI, and I will add this feature.

With nuxi, I supposed it's just a matter of making the function that creates the module recursive, and splitting the arguments after `nuxi module add` by the spaces as an array, and having a progress of the modules being added like *Installing module 1 of 3* and finally display the list of all installed modules.

Sounds fun! 🔥💚😊 

Nuxt is so cool, modern, and elegant!!! Thanks to all the good work being done by the team.

**Comment by nobkd** [comment]
Duplicate of #65

**Reply by angelhdzmultimedia** [reply]
Thank you for the find! 😊🔥💚
